### DISCUSSION TITLE
How to append trait manager to multiple selectors (#id)

### DISCUSSION THREAD

**Discussion by echobinod** [GrapesJS/grapesjs#3543] [Q&A]
I can append trait manager to only one selector this way:

```
traitManager: {
        appendTo: "#menu",
      }
```
I want to append it to multiple id's (#menu and #section)

How can I do that ?

(this does'nt work)

```
traitManager: {
        appendTo: "#menu, #section",
      }
```

**Comment by artf** [comment]
At the moment is not possible, the trait manager is rendered only in a single container.
